x-pack/test/saved_object_api_integration/common/suites/import.ts

```diff
@@ -76,32 +152,66 @@ export function importTestSuiteFactory(es: any, esArchiver: any, supertest: Supe
           expect(object!.error).to.eql({ type: 'unsupported_type' });
         } else {
           // 409
-          expect(object!.error).to.eql({ type: 'conflict' });
+          let error: Record<string, unknown> = {
+            type: 'conflict',
+            ...(expectedNewId && { destinationId: expectedNewId }),
+          };
+          if (fail409Param === 'ambiguous_conflict_1a1b') {
+            // "ambiguous source" conflict
+            error = {
+              type: 'ambiguous_conflict',
+              destinations: [getConflictDest(`${CID}1`)],
+            };
+          } else if (fail409Param === 'ambiguous_conflict_2c') {
+            // "ambiguous destination" conflict
+            error = {
+              type: 'ambiguous_conflict',
+              destinations: [getConflictDest(`${CID}2a`), getConflictDest(`${CID}2b`)],
+            };
+          } else if (fail409Param === 'ambiguous_conflict_2c2d') {
+            // "ambiguous source and destination" conflict
+            error = {
```

Comment:
nit: you can optionally collapse this with the `ambiguous_conflict_2c` case above. They have the same expected error.

Reply:
Ah, actually it's not used at all anymore since we suppress these types of conflicts on the server side. I'll remove it.